Assets/_Laz/Scripts/Environment/Obstacle.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+namespace Laz
+{
+    [RequireComponent(typeof(Collider))]
+    public class Obstacle : MonoBehaviour
+    {
+        private void OnCollisionEnter(Collision other)
+        {
+            if (other.collider.CompareTag(Tags.LazPlayer))
+            {
+                LazCoordinatorBehaviour behaviour = other.gameObject.GetComponent<LazCoordinatorBehaviour>();
```

Comment:
I think you can just do `other.GetComponent`

Reply:
Nah doesn't seem possible with the Collision class